foundation/foundation-upgrade/src/main/java/com/boozallen/aissemble/upgrade/migration/v1_11_0/DockerModulePomDependencyTypeMigration.java

```diff
@@ -0,0 +1,110 @@
+package com.boozallen.aissemble.upgrade.migration.v1_11_0;
+
+/*-
+ * #%L
+ * aiSSEMBLE::Foundation::Upgrade
+ * %%
+ * Copyright (C) 2021 Booz Allen
+ * %%
+ * This software package is licensed under the Booz Allen Public License. All Rights Reserved.
+ * #L%
+ */
+
+import com.boozallen.aissemble.upgrade.migration.AbstractPomMigration;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.maven.model.Dependency;
+import org.apache.maven.model.Model;
+import org.technologybrewery.baton.util.pom.PomHelper;
+import org.technologybrewery.baton.util.pom.PomModifications;
+
+import java.io.File;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static org.technologybrewery.baton.util.pom.LocationAwareMavenReader.END;
+import static org.technologybrewery.baton.util.pom.LocationAwareMavenReader.START;
+
+/**
+ * This migration fixes the build cache dependency check by removing the dependency `pom` type for the java pipeline
+ * and replace the dependency type with `habushu` if it's python pipeline
+ */
+public class DockerModulePomDependencyTypeMigration extends AbstractPomMigration {
```

Comment:
Don't forget to add these migrations to the Release Notes table. Think we're missing one from #492 as well.